grafana/files/grafana.ini

```diff
@@ -210,8 +210,12 @@ enabled = false
 
 #################################### Auth LDAP ##########################
 [auth.ldap]
-;enabled = false
-;config_file = /etc/grafana/ldap.toml
+{%- if server.auth.engine == 'ldap' %}
```

Comment:
One problem is that it can't support both basic and ldap authentications at the same time. IIUC this will be an issue for the Grafana client state since it uses basic auth.